### PR TITLE
K26F HIC and FRDM-K32W042 board

### PR DESCRIPTION
**Pull request by flit**
This PR adds a new NXP K26F HIC. It has high speed USB and SWO support enabled. The FRDM-K32W042 board is also added that uses this HIC.

### CODE REVIEW

File: tools/post_build_script.py [hunk 1]
```diff
@@ -173,6 +173,10 @@ def post_build_script(input_file, output_file, board_id=None, family_id=None, bi
         offset_update.create_padded_image(output_file_legacy,
                                           output_file_legacy_5000,
                                           start, pad_addr, 0x40)
+    elif start == 0 or start == 0x80000 or start == 0x20000:
+        pass
+    else:
+        assert 0, "Unsupported start address 0x%x" % start
```
Comment (brianesquilona) [inline tools/post_build_script.py]: lpc4322 has a start address of 0x1A010000, so the build script failed

Comment (flit) [reply tools/post_build_script.py]: Ah, thanks for the explanation!

**Pull request merged**